core/io.rbs

```diff
@@ -3251,6 +3251,10 @@ class IO < Object
   #     f.close
   #
   alias to_i fileno
+
+  interface _ForFd[RET]
+    def for_fd: (?) -> RET
```

Comment:
The argument is rough.

Actually, the definition should be the following:


```rbs
interface _ForFd[MODE, RET]
  def for_fd: (Integer fd, ?MODE) -> RET
end
```

But the definition is wrong because the following class does not match the interface.

```rbs
class ForFd
  def self.for_fd: (Integer fd) -> instance
end
```